CHANGES.rst

```diff
@@ -29,6 +29,11 @@ ramp_fitting
 - Updated the argument description and parameter definition for `maximum_cores`
   to accept integer values to be passed to STCAL ramp_fit.py. [#8123]
 
+tweakreg
```

Comment:
Need to rebase the PR from jwst/master, because the CHANGES.rst file has now been updated for the 1.13.3 release, and then also move this entry to the top (unreleased) section of the updated file.

Reply:
Done. Thanks.